### ISSUE TITLE
Depositor obtains BaseWrapper balance on pull deposit

### ISSUE THREAD

**Issue by skeletor-spaceman** [yearn/yearn-vaults#533]
afterBal should be changed to deposited
https://github.com/yearn/yearn-vaults/blob/b3f33bd5a8ece52b6504d0c1cd9029fb639d6349/contracts/BaseWrapper.sol#L182

also, pull boolean should be immutable, not sure there is a usecase for both pull and non-pull deposits

**Comment by westonnelson** [comment]
I agree this is a bit confusing: perhaps using updatedBal or depositorBal might make this even more clear.
